Tests/MongoSwiftTests/TestUtils.swift

```diff
@@ -14,7 +14,7 @@ class MongoSwiftTestCase: XCTestCase {
 
     /// Gets the name of the database the test case is running against.
     internal class var testDatabase: String {
-        return "test"
+        return String(describing: self)
```

Comment:
> It also changes the database used for each case to simply be the name of the test case, instead of "test" for everything.

On a related note to https://github.com/mongodb/mongo-swift-driver/pull/181/files#r244839841, I'm not sure if using different database names across the test suite is an improvement. There is definitely more overhead to creating a database than a collection, as each database has its own file(s). Consolidating the test suite to a single database name when possible (some spec tests may require their own names) makes it easier to clean up data that is left behind. Looking again to the PHP library, which inspired this request, we use "test" by default but allow the user to override this with an environment variable or test config [option](https://github.com/mongodb/mongo-php-library/blob/master/phpunit.xml.dist#L20). I'm not sure if the latter exists in Swift, since we're using XCTest here, but I'd think an environment variable is possible.

Reply:
It seems I misunderstood this line from the ticket description as well. I took "the test suite" to mean a single test file, e.g. `DocumentTests`, hence this change. I'll simply revert it, and if we want to add test environment configuration I can follow up in another ticket.